live/src/config.sh

```diff
@@ -12,10 +12,21 @@ echo "Configure image: [$kiwi_iname]..."
 # setup baseproduct link
 suseSetupProduct
 
+# enable the corresponding repository
+DISTRO=$(grep "^NAME" /etc/os-release | cut -f2 -d\= | tr -d '"' | tr " " "_")
+REPO="/etc/zypp/repos.d/agama-${DISTRO}.repo"
+if [ -f "${REPO}.disabled" ]; then
+  mv "${REPO}.disabled" $REPO
+fi
+rm /etc/zypp/repos.d/*.disabled
+
 # configure the repositories in the Live system
 # import the OBS key for the systemsmanagement OBS project
 rpm --import /tmp/systemsmanagement_key.gpg
 rm /tmp/systemsmanagement_key.gpg
+# import the OBS key for the Devel:YaST:Agama:Head project
```

Comment:
At first I was confused here, then I realized it is the internal OBS (IBS).  Maybe it should be mentioned in the comment.